pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-7713.xml

```diff
@@ -52,6 +52,10 @@
                 <constraints primaryKey="true" primaryKeyName="block_tabular_question_pk"/>
             </column>
 
+            <column name="column_span" type="bigint">
+                <constraints nullable="false" />
+            </column>
+
```

Comment:
Since this liquibase deploy only made upto dev : My idea is to cleanup these tabes in Dev manually and pretend this liquibase script was never deployed. Tried it locally and seemed to work